src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/PortScanCommand.java

```diff
@@ -20,6 +21,8 @@
 @Singleton
 class PortScanCommand implements SerializableCommand {
```

Comment:
Maybe this command should be renamed now, since it has more ways to detect targets than port scanning. `scan-targets` maybe? We can then do some refactoring to have some pluggable and configurable interface for various detection methods.

Reply:
Sure, I've done this. I'll see if I can do this on the container-jfr-web side of things too.